google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/facet/AppEngineSupportProvider.java

```diff
@@ -307,6 +309,11 @@ public void addSupport(@NotNull Module module,
       AppEngineSupportProvider.this
           .addSupport(module, rootModel, myFrameworkSupportModel,
               PersistenceApiComboboxUtil.getSelectedApi(myPersistenceApiComboBox));
+
+      // Called on New Project or New Module dialogs.
+      UsageTrackerProvider.getInstance()
+          .trackEvent(GctTracking.APP_ENGINE_ADD_SUPPORT)
```

Comment:
Shouldn't this be APP_ENGINE_STANDARD_NEW_PROJECT_WIZARD (or something more concise)? I was under the impression that this is called on new projects, but I may be confusing things now.